<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsImage</class>
 <widget class="QWidget" name="SettingsImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../img.qrc">
    <normaloff>:/image</normaloff>:/image</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Frame</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Mask</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="framePage">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="FrameGroupBox">
         <property name="title">
          <string>Frame:</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Border width:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="frameWidth">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Border value:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="frameValue">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="maskPage">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="MasquradingGroupBox">
         <property name="title">
          <string>Masqurading pixel:</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Value</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="maskValue">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="maskAfter">
            <property name="text">
             <string>after</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="maskBefore">
            <property name="text">
             <string>before</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>New value:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="maskNewValue">
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Color:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="maskColor">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAccepted">
   <property name="text">
    <string>Accepted</string>
   </property>
   <property name="shortcut">
    <string>Enter</string>
   </property>
  </action>
  <action name="actionReject">
   <property name="text">
    <string>Reject</string>
   </property>
   <property name="toolTip">
    <string>Reject</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionsetColor">
   <property name="text">
    <string>setColor</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsImage</receiver>
   <slot>on_actionAccepted_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsImage</receiver>
   <slot>on_actionReject_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maskColor</sender>
   <signal>clicked()</signal>
   <receiver>SettingsImage</receiver>
   <slot>on_actionsetColor_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_actionAccepted_triggered()</slot>
  <slot>on_actionReject_triggered()</slot>
  <slot>on_actionsetColor_triggered()</slot>
 </slots>
</ui>
